identity/src/keypair.rs

```diff
@@ -494,12 +508,24 @@ impl PublicKey {
     /// Encode the public key into a protobuf structure for storage or
     /// exchange with other nodes.
     #[deprecated(note = "Renamed to `PublicKey::encode_protobuf`.")]
+    #[cfg(any(
+        feature = "ecdsa",
+        feature = "secp256k1",
+        feature = "ed25519",
+        feature = "rsa"
+    ))]
```

Comment:
We want all the public functions to always be present! The goal is that the user doesn't have to deal with any `cfg`, regardless which features are activated.

Reply:
> about what to return

If none of the features are enabled, it is impossible to construct a `Keypair` and a `Publickey`, hence we can use `unreachable!` in that case!

Reply:
@thomaseizinger Done